spec/requests/socure_webhook_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'SocureWebhooks', type: :request do
+  describe 'POST /api/webhooks/socure/event' do
+    it 'returns OK' do
+      post '/api/webhooks/socure/event'
+      expect(response).to have_http_status(:ok)
+    end
+  end
+end
```

Comment:
request specs are generally more "expensive" than controller specs, they spin up more of the middleware stack. unless there is some specific middleware behavior we're intending to implement + test, I'd recomomend removing in favor of just controller specs

Reply:
Replaced with a controller spec.